orchagent/bulker.h

```diff
@@ -280,29 +282,19 @@ class EntityBulker
                 {
                     rs.push_back(entry);
                 }
-            }
-            size_t count = rs.size();
-            std::vector<sai_status_t> statuses(count);
-            sai_status_t status = (*remove_entries)((uint32_t)count, rs.data(), SAI_BULK_OP_ERROR_MODE_IGNORE_ERROR, statuses.data());
-            if (status == SAI_STATUS_SUCCESS)
-            {
-                SWSS_LOG_INFO("EntityBulker.flush removing_entries %zu\n", removing_entries.size());
-            }
-            else
-            {
-                SWSS_LOG_ERROR("EntityBulker.flush remove entries failed, number of entries to remove: %zu, status: %s",
-                                removing_entries.size(), sai_serialize_status(status).c_str());
-            }
 
-            for (size_t ir = 0; ir < count; ir++)
-            {
-                auto& entry = rs[ir];
-                sai_status_t *object_status = removing_entries[entry];
-                if (object_status)
+                if (rs.size() >= MAX_BULK_SIZE)
```

Comment:
>if [](http://example.com/codeflow?start=16&length=2)

You can move this if-block into inside of above if-block

Reply:
Moved it to the if-block above.